analytics_data_api/v0/tests/views/test_courses.py

```diff
@@ -652,6 +652,90 @@ def test_get_404(self):
         self.assertEquals(response.status_code, 404)
 
 
+class CourseUsersProblemDataTests(DemoCourseMixin, TestCaseWithAuthentication):
+    def _get_data(self, user_id=None):
+        return self.authenticated_get('/api/v0/courses/{}/users/{}/problem_data/'.format(self.course_id, user_id))
+
+    def test_get_problem_data(self):
+        user_id = 111
+        G(
+            models.UserProblemWeeklyData,
+            id=1,
+            week_ending="2015-08-20",
```

Comment:
Hmm, well somewhere some code is translating the dates to ISO8601 strings, since that's what I see in the JSON response... Perhaps you should change the serializer to do that, using `week_ending = serializers.DateTimeField(format=settings.DATETIME_FORMAT)` as is done in other serializers like `CourseActivityWeeklySerializer`? Otherwise you should probably change the before and after test to use `some_date_obj.strftime(settings.DATETIME_FORMAT)` as you can see other tests doing.